### PR TITLE
Accepted file can also be Blob

### PR DESCRIPTION
**Pull request by patrickdet**
According to the specification https://developer.mozilla.org/en-US/docs/Web/API/FileReader a FileReader can read objects of the type File or Blob.

This PR adds the ability to also read in Blobs.

I ran into a Problem using the library where i needed to read in a blob but the methods only accept File objects. First I thought I could create a file like `new File([myblob], "filename")` but that way the mimetype info gets lost and when i call `readAsDataUrl()` i get a string but without the proper mimetype it's useless. This PR fixes all of that by allowing Blobs as well.

### CODE REVIEW

File: PromiseFileReader.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 function readAsDataURL(file) {
-  if (!(file instanceof File)) {
-    throw new TypeError('Must be a File')
+  if (!(file instanceof File || file instanceof Blob)) {
```
Comment (jahredhope) [inline PromiseFileReader.js]: I don't believe `file instanceof File` is needed anymore.

A File inherits Blob. The second statement will always be true if the first statement is true.

https://developer.mozilla.org/en-US/docs/Web/API/File

Comment (patrickdet) [inline PromiseFileReader.js]: ahh true! i will fix that quickly

**Comment by jahredhope** [comment]
Awesome work!

**Comment by patrickdet** [comment]
@jahredhope thanks for this library!. It's tiny but I am using this FileReader pattern a lot. With your library + async/await it makes the code so much more readable

**Pull request merged**